packages/kotti-ui/source/kotti-filter/components/FilterButtonLink.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div class="kt-filter-button-link">
+		<span @click="handleClick">
+			<i class="yoco" v-text="icon" /> {{ label }}
+		</span>
+	</div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api'
+
+export default defineComponent<{ label: string; icon: string }>({
+	name: 'FilterButtonLink',
+	props: {
+		label: {
+			type: String,
+			default: '',
```

Comment:
If this is not required, it should default to `null`. However, it kinda looks required to me